project/src/components/map/map.tsx

```diff
@@ -1,9 +1,10 @@
 import { useEffect, useRef } from 'react';
-import { Icon, Marker } from 'leaflet';
+import L, { Icon, Marker } from 'leaflet';
```

Comment:
сокращения запрещены

```suggestion
import Leaflet, { Icon, Marker } from 'leaflet';
```